### PR TITLE
updated the software registry endpoints

### LINKED ISSUE #23: updated the software registry endpoints

### PR DESCRIPTION
**Pull request by yuusufisse**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -805,11 +805,17 @@ components:
...
+        - DEPRECATED
         - DECLINED
 
     SoftwareRegistry:
       type: object
+      required:
+        - name
+        - description
+        - url
+        - image
```
Comment (Jdallos) [inline swagger.yaml:817]: Are you sure image should be required? Probably most things have an image but it seems to create a bit more work for the users- unless you are able to get the image straight from the Url (which would be very cool).

File: swagger.yaml [hunk 2]
```diff
@@ -844,4 +850,16 @@ components:
         lastUpdatedAt:
           type: string
           format: date-time
-          description: Date and time when the software entry was last updated
\ No newline at end of file
+          description: Date and time when the software entry was last updated
+        recommend:
```
Comment (Jdallos) [inline swagger.yaml:854]: Needs description

File: swagger.yaml [hunk 3]
```diff
@@ -844,4 +850,16 @@ components:
...
+          type: array
+          items:
+            type: string
+        tags:
```
Comment (Jdallos) [inline swagger.yaml:859]: Needs description

File: swagger.yaml [hunk 4]
```diff
@@ -844,4 +850,16 @@ components:
...
+          type: array
+          items:
+            type: string
+        users:
```
Comment (Jdallos) [inline swagger.yaml:864]: Needs description

**Review by Jdallos — CHANGES_REQUESTED** [review]

File: swagger.yaml (hunk 1 above)
Comment (yuusufisse) [reply swagger.yaml:817]: image is required for the logo of the software

**Review by Jdallos — APPROVED** [review]

**Pull request merged**